impl/src/rendering/bazel.rs

```diff
@@ -220,18 +219,32 @@ impl BuildRenderer for BazelRenderer {
       })
     }
 
-    let build_file_path = path_prefix.as_path().join(buildfile_suffix);
-    let rendered_alias_build_file = self
-      .render_aliases(&workspace_context, &crate_contexts)
-      .map_err(|e| RazeError::Rendering {
-        crate_name_opt: None,
-        message: unwind_tera_error!(e),
-      })?;
+    for member_path in planned_build.workspace_context.workspace_members.iter() {
+      let all_packages: Vec<CrateContext> = planned_build
+        .crate_contexts
+        .iter()
+        .filter(|ctx| {
+          ctx.is_binary_dependency || ctx.workspace_member_dependents.contains(member_path)
+        })
+        .map(|ctx| ctx.clone())
+        .collect();
+      let rendered_alias_build_file = self
```

Comment:
Nit: I would put a line break between the two bindings to avoid everything becoming a wall of text over time.

Reply:
Done